pkg/volume/flexvolume/flexvolume.go

```diff
@@ -301,8 +301,8 @@ func (f *flexVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 			options = append(options, "rw")
 		}
 		// Extract secret and pass it as options.
-		if f.secret != "" {
-			options = append(options, "secret="+f.secret)
+		for name, secret := range f.secrets {
+			f.options[optionKeySecret+"/"+name] = secret
```

Comment:
Sounds good to me. Will put base64 encoding. It is the format we are using elsewhere in Kubernetes, no reason to do it differently.